### PR TITLE
Add sorting options to the page. Allow to add more options in the future

### LINKED ISSUE #129: ✨ Filter issues by no replies
## Description
It would be cool if there was a way to filter the issues so that we could find issues that don't have any replies on them.

### PR DESCRIPTION
**Pull request by vanya829**
# Description
What did you change? Please leave a breif description of what's changed.
Added extra param to the request, to allow issues sorting
Added two sorting buttons to the main page
# Screenshots
If applicable, please leave some screenshots showing the changes you've made.

# Issues

> Closes #129

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,6 +41,20 @@ <h1 class="text-5xl font-bold text-primary uppercase">
           </button>
         </div>
       </header>
+      <div>
+        <button 
+        class="text-primary text-lg font-semibold uppercase"
```
Comment (damcclean) [inline index.html]: Would you mind putting these buttons beside the language toggle (I'll need to work something out about spacing but I'll do that later)

File: index.html [hunk 2]
```diff
@@ -96,7 +111,7 @@ <h3 class="text-center text-blue mb-4 break-words">{{ result.user.login }}/{{ re
       </div>
     </footer>
 
-    <script src="/build/app.js"></script>
+    <script src="./build/app.js"></script>
```
Comment (damcclean) [inline index.html]: would you mind changing that back to how I had it? (just my personal preference)

File: index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <title>Hacktoberfest Finder</title>
-    <link rel="stylesheet" href="/build/app.css" />
+    <link rel="stylesheet" href="./build/app.css" />
```
Comment (damcclean) [inline index.html]: would you mind changing that back to how I had it? (just my personal preference)

File: index.html [hunk 4]
```diff
@@ -15,7 +15,7 @@
     <meta property="og:image" content="" />
     <meta property="og:url" content="https://hacktoberfest-finder.netlify.com/" />
 
-    <link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicons/apple-touch-icon.png" />
+    <link rel="apple-touch-icon" sizes="180x180" href="../assets/images/favicons/apple-touch-icon.png" />
```
Comment (damcclean) [inline index.html]: don't know why you did this but would you mind removing the double dots at the start of this url.

File: index.html (hunk 1 above)
Comment (vanya829) [reply index.html]: Changed sorting buttons to filter. Moved to the same level as language filter

File: index.html (hunk 2 above)
Comment (vanya829) [reply index.html]: Revert all such changes. Committed them first time by mistake.

File: index.html (hunk 3 above)
Comment (vanya829) [reply index.html]: https://github.com/damcclean/hacktoberfest-finder/pull/134#discussion_r330713813

File: index.html (hunk 4 above)
Comment (vanya829) [reply index.html]: https://github.com/damcclean/hacktoberfest-finder/pull/134#discussion_r330713813

**Comment by damcclean** [comment]
All looks good! Would you mind adding yourself to the contributors file.

**Pull request merged**